apps/meteor/tests/unit/app/livechat/server/lib/isMessageFromBot.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
+import p from 'proxyquire';
+import sinon from 'sinon';
+
+import { createFakeMessage, createFakeUser } from '../../../../../mocks/data';
+
+const modelsMock = {
+	Users: {
+		findOneById: sinon.stub(),
+	},
+};
+
+const { isMessageFromBot } = p.noCallThru().load('../../../../../../app/livechat/server/lib/isMessageFromBot', {
+	'@rocket.chat/models': modelsMock,
+});
+
+describe('isMessageFromBot', () => {
```

Comment:
Please add a test case for when `findOneById` returns `undefined` (user doesn't exist)